engine/build.gradle

```diff
@@ -11,6 +11,15 @@ dependencies {
     compile "org.terasology:gestalt-asset-core:6.0.0"
     compile "org.terasology:gestalt-entity-system:6.0.0"
 
+
+    //TODO inserted by someone who has no idea how the gradle thing works. Inspect whether necessary (for tests)
```

Comment:
Oh I see. This block was copied from the desktop `build.gradle` right? Because if you're running a unit test directly against the engine you wouldn't get the dependencies declared for the desktop.

If you only need them for tests you can use `testCompile` instead of `compile` and they'll only matter when execution is aimed at testing, not running normally. That might also sidestep any issues introduced for *Android* suddenly also depending directly on these for compilation. 

Although I don't know how it would impact a testing context for Android but ... there aren't any Android tests yet, anyway. So maybe just move to `testCompile` and update the comment/todo with some of these notes and a future consideration to update it further